lib/utmp.c

```diff
@@ -151,8 +151,7 @@ failtmp(const char *username, const struct utmpx *failent)
 static /*@null@*/ /*@only@*/struct utmpx *
 get_current_utmp(void)
 {
-	struct utmpx  *ut;
-	struct utmpx  *ret = NULL;
+	struct utmpx *ut;
```

Comment:
Please keep the two spaces after the type.

Reply:
It would be nice to copy at least this short explanation to some file/document.